components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -120,12 +129,12 @@ const UpdateOrderPopUp = ({ setMenuState, contribution, createNotification, setS
         key: `tier-${tier.id}`,
         title: tier.name,
         flexible: tier.amountType === 'FLEXIBLE' ? true : false,
-        amount: tier.amountType === 'FLEXIBLE' ? tier.minimumAmount : tier.amount,
+        amount: tier.amountType === 'FLEXIBLE' ? tier.minimumAmount.valueInCents : tier.amount.valueInCents,
         id: tier.id,
-        currency: tier.currency,
+        currency: tier.amount.currency,
         interval: tier.interval,
         presets: tier.presets,
-        minimumAmount: tier.amountType === 'FLEXIBLE' ? tier.minimumAmount : 100,
+        minimumAmount: tier.amountType === 'FLEXIBLE' ? tier.minimumAmount.valueInCents : 100,
       }));
     tierOptions.unshift(customTierOption);
```

Comment:
**[:hammer_and_wrench: **Important**]** It's not from this PR, but I just realized we need to make sure that `collective.settings.disableCustomContributions` is not true before adding the "custom tier" option to the list (see https://github.com/opencollective/opencollective-api/pull/3936).

Can be done in another PR if more convenient.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Good shout, I didn't realise that was a thing!

Reply:
No big deal, added to this PR :)